EIPS/eip-3675.md

```diff
@@ -177,6 +175,14 @@ Existing rewards for producing and sealing blocks are deprecated along with the
 
 The fork choice rule of the PoW mechanism becomes completely irrelevant after the upgrade and is replaced with the corresponding rule of the new PoS consensus mechanism.
 
+### Remove of `POS_CONSENSUS_VALIDATED`
+
+In prior draft versions of this EIP, an additional POS event -- `POS_CONSENSUS_VALIDATED` -- was required as a validation condition for blocks. This event gave the signal to either fully incorporate or prune the block from the block tree.
+
+This event was removed for two reasons:
+1. This event was an unnecessary optimization to allow for pruning of "bad" blocks from the block tree. This optimization was unnecessary because the PoS consensus would never send `POS_FORKCHOICE_UPDATED` for any such bad blocks or their descendants, and eventually any such blocks would be able to be pruned after a PoS finality event of an alternative branch in the block tree.
+2. This event was dangerous in some scenarios because a block could be referenced by two *different* and conflicting PoS branches. Thus for the same block in some scenarios, both a `POS_CONSENSUS_VALIDATED == True` and `POS_CONSENSUS_VALIDATED == FALSE` event could sent, entirely negating the ability to safely perform the optimization in (1).
```

Comment:
Nit: 
> `POS_CONSENSUS_VALIDATED == True` and `POS_CONSENSUS_VALIDATED == FALSE`

Should be `POS_CONSENSUS_VALIDATED == TRUE` and `POS_CONSENSUS_VALIDATED == FALSE` -- we can fix the inconsistency of the case between `True` and `FALSE` during the next update

Reply:
whoops! I'll submit a tiny PR